core/blockly.js

```diff
@@ -141,36 +141,30 @@ Blockly.svgSize = function(svg) {
 };
 
 /**
- * Schedule a call to the resize handler.  Groups of simultaneous events (e.g.
- * a tree of blocks being deleted) are merged into one call.
- * @param {Blockly.WorkspaceSvg} workspace Any workspace in the SVG.
+ * Size the workspace when the contents change.  This also updates
+ * scrollbars accordingly.
+ * @param {!Blockly.WorkspaceSvg} opt_workspace. The svg group of the workspace
+ *   to resize. If the workspace isn't specified, the main workspace will be
+ *   used.
  */
-Blockly.asyncSvgResize = function(workspace) {
-  if (Blockly.svgResizePending_) {
-    return;
-  }
+Blockly.resizeSvgContents = function(opt_workspace) {
+  var workspace = opt_workspace;
   if (!workspace) {
```

Comment:
PTAL

I went with the "other" approach. I made resizeSvgContents's workspace required and tracked down the callers that sent null workspaces. I don't think they were intending to so I fixed them to send real workspaces. These changes are easier to see if you look at the most recent commit.